ui/lib/config-ui/addon/components/login-settings/page/details.hbs

```diff
@@ -0,0 +1,67 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+{{#if @rule}}
+  <PageHeader as |p|>
+    <p.top>
+      <Page::Breadcrumbs @breadcrumbs={{@breadcrumbs}} />
+    </p.top>
+    <p.levelLeft>
+      <h1 class="title is-3">
+        {{@rule.name}}
+      </h1>
+    </p.levelLeft>
+  </PageHeader>
+
+  <nav class="tabs" aria-label="navigation for rule details">
+    <ul>
+      <li>
+        <LinkTo @route="login-settings.rule.details" @model={{@rule}} data-test-rule-tab>
```

Comment:
Selectors are on my 🧠 because I'm doing some organizing of them. All that to say, get ready for me to become the annoying selector 🕵🏻‍♀️. If there is a tab, let's use the tab [GENERAL](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.ts#L16) selector.
```suggestion
        <LinkTo @route="login-settings.rule.details" @model={{@rule}} data-test-tab="rule">
```